src/test/kotlin/TestUtils.kt

```diff
@@ -43,4 +42,106 @@ object TestUtils {
                 unitPositions
         )
     }
+
+    fun getMockedBasicActions(): Map<String, ActionOption> {
+        val halfAimMock = Mockito.mock(HalfAim::class.java)
+        Mockito.`when`(halfAimMock.actionCost).thenReturn(ActionCost.HALF_ACTION)
```

Comment:
[Minor] Doing a static import of Mockito.when will cut a lot of this noise.

Reply:
Fixed in TestUtils